setup.py

```diff
@@ -20,6 +20,7 @@
     'install_requires': [
         'torch-nightly',
         'lark-parser',
+        'pkbar',
```

Comment:
No no no, the dependency of TorchANI should be kept a minimum. Previously I intentionally not depending on `ignite` and `tqdm`. The rule is, if the user installed it, it is nice to use, if not, just disable the corresponding function.

Reply:
Adding dependencies to `test_requires` is OK, but don't add anything to `install_requires`.